addons/binding/org.openhab.binding.tankerkoenig/src/main/java/org/openhab/binding/tankerkoenig/handler/BridgeHandler.java

```diff
@@ -0,0 +1,216 @@
+package org.openhab.binding.tankerkoenig.handler;
```

Comment:
Something general: please reduce the number of empty lines. The code contains several empty lines that are not necessary for the readability of the code.